### PR TITLE
Remove manual logger imports

### PR DESCRIPTION
**Pull request by cjyetman**

### CODE REVIEW

**Comment by AlexAxthelm** [comment]
Move to close this PR. The current behavior (importing the log_*` functions) is the preferred mechanism for logging from packages, since `logger` can assign the namespace correctly, which allows for better control over logging messages:
```r
logger::log_threshold("WARN") # define global logging level
logger::log_threshold("DEBUG", namespace = "pacta.data.scraping") # get detailed messages from only this package
```

**Comment by AlexAxthelm** [comment]
The starting point is https://daroczig.github.io/logger/articles/r_packages.html (with references to https://daroczig.github.io/logger/articles/customize_logger.html#log-namespaces),

That page includes:

> By default, if you are calling the [?log_level](https://daroczig.github.io/logger/reference/log_level.html) function family from an R package after importing from the logger package, then logger will try to auto-guess the calling R package name and use that as the default namespace

but the actual behavior was discovered through a bit of experimentation, and looking at the demo package that is included (https://github.com/daroczig/logger/tree/master/inst/demo-packages/logger-tester-package).

Basically, the namespace for the log entry is whatever package's version of `log_*` is being used, so if you import `log_*`,  and call it within a package (un-namespaced), it uses that packages' imported logging, but if you use `logger::log_*` then it uses `logger`'s (rather than `pacta.data.scraping`) and assigns the log to the `global` namespace.

**Comment by cjyetman** [comment]
gonna close this and rather consider replacing {logger} with something that has more standard behavior
